### PR TITLE
Gilad/feedback system/feedbackdao1

### PR DESCRIPTION
**Pull request by gilRoz23**

### CODE REVIEW

File: src/main/kotlin/com/hibob/academy/feedbacks_system/dao/models.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    OPERATIONS,
+    RESEARCH_AND_DEVELOPMENT,
+    CUSTOMER_SERVICE,
+    LEGAL
+}
+
+
+data class FeedbackData(
+    val id: Long,
+    val company_id: Long,
```
Comment (vasofi) [inline src/main/kotlin/com/hibob/academy/feedbacks_system/dao/models.kt:20]: use the java standard, which is camel case companyId

File: src/main/kotlin/com/hibob/academy/feedbacks_system/dao/models.kt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    val content: String,
+    val isAnonymous: Boolean,
+    val status: Boolean,
+    val feedbackProviderId: Long?,
```
Comment (vasofi) [inline src/main/kotlin/com/hibob/academy/feedbacks_system/dao/models.kt:24]: note that this is not nullable in the table that you created. One of them needs to be changed

File: src/main/kotlin/com/hibob/academy/feedbacks_system/dao/models.kt [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package com.hibob.academy.feedbacks_system.dao
```
[the remark this answers is not in the document]
Comment (gilRoz23) [reply src/main/kotlin/com/hibob/academy/feedbacks_system/dao/models.kt:1]: thank you, I will change it

File: src/main/kotlin/com/hibob/academy/feedbacks_system/dao/models.kt (hunk 2 above)
Comment (gilRoz23) [reply src/main/kotlin/com/hibob/academy/feedbacks_system/dao/models.kt:24]: I think it is. as long as it's not mentioned "not null" it is nullable, isn't it like that?

**Review by vasofi — APPROVED** [review]
decide what to do with feedbackProviderId

**Pull request merged**